x/delayedack/keeper/rollapp_packets_list_filter.go

```diff
@@ -0,0 +1,58 @@
+package keeper
+
+import commontypes "github.com/dymensionxyz/dymension/v3/x/common/types"
+
+type filterFunc func(val commontypes.RollappPacket) bool
+
+type rollappPacketListFilter struct {
+	prefixes        [][]byte
```

Comment:
why u need to support multiple prefixes?
IMO it's more clearer to leave `rollappPacketListFilter` for single query

Reply:
for filtering for both `REVERTED` and `FINALIZED` with one filter